packages/client-core/src/components/UserMediaWindow/index.tsx

```diff
@@ -68,6 +72,30 @@ interface Props {
   type: 'screen' | 'cam'
 }
 
+const useDrawMocapLandmarks = (
+  videoElement: HTMLVideoElement,
+  canvasCtxRef: React.MutableRefObject<CanvasRenderingContext2D | undefined>,
+  canvasRef: RefObject<HTMLCanvasElement>,
+  peerID: PeerID
+) => {
+  let lastTimestamp = 0
+
+  useExecute(
+    () => {
+      if (videoElement.paused || videoElement.ended || !videoElement.currentTime) return
+      const mocapBuffer = timeSeriesMocapData.get(peerID)
+      if (mocapBuffer) {
+        const lastMocapResult = mocapBuffer.getLast()
+        if (lastMocapResult && lastMocapResult.timestamp !== lastTimestamp) {
+          lastTimestamp = lastMocapResult.timestamp
+          drawPoseToCanvas(canvasCtxRef, canvasRef, lastMocapResult.results.poseLandmarks)
+        }
+      }
+    },
+    { after: MotionCaptureSystem }
```

Comment:
This should be after the presentation group. Anything not critical to rendering the 3d world should happen after we have rendered the scene